pallets/claims/src/mock.rs

```diff
@@ -0,0 +1,129 @@
+use crate::{Config, EthereumAddress, GenesisConfig, Module};
+use frame_support::{impl_outer_event, impl_outer_origin, parameter_types};
+use frame_system;
+use orml_traits::parameter_type_with_key;
+use primitives::{AssetId, Balance};
+use sp_core::H256;
+use sp_runtime::{
+	testing::Header,
+	traits::{BlakeTwo256, IdentityLookup, Zero},
+};
+
+impl_outer_origin! {
+	pub enum Origin for Test {}
+}
+
+#[derive(Clone, Eq, PartialEq)]
+pub struct Test;
+
+parameter_types! {
+	pub const BlockHashCount: u64 = 250;
+}
+
+impl frame_system::Config for Test {
+	type BaseCallFilter = ();
+	type BlockWeights = ();
+	type BlockLength = ();
+	type Origin = Origin;
+	type Call = ();
+	type Index = u64;
+	type BlockNumber = u64;
+	type Hash = H256;
+	type Hashing = BlakeTwo256;
+	type AccountId = u64;
+	type Lookup = IdentityLookup<Self::AccountId>;
+	type Header = Header;
+	type Event = ();
+	type BlockHashCount = BlockHashCount;
+	type DbWeight = ();
+	type Version = ();
+	type PalletInfo = ();
+	type AccountData = ();
+	type OnNewAccount = ();
+	type OnKilledAccount = ();
+	type SystemWeightInfo = ();
+}
+
+pub type Amount = i128;
+
+parameter_type_with_key! {
+	pub ExistentialDeposits: |currency_id: AssetId| -> Balance {
+		Zero::zero()
+	};
+}
+
+impl orml_tokens::Config for Test {
+	type Event = ();
+	type Balance = Balance;
+	type Amount = Amount;
+	type CurrencyId = AssetId;
+	type WeightInfo = ();
+	type ExistentialDeposits = ExistentialDeposits;
+	type OnDust = ();
+}
+
+pub type Currency = orml_tokens::Module<Test>;
+
+parameter_types! {
+	pub Prefix: &'static [u8] = b"I hereby claim all my xHDX tokens to wallet:";
```

Comment:
"HDX tokens to wallet: "?

Reply:
The whitespace could do more harm than good. I agree with change to HDX.

Reply:
any specific reason not to add space after : ? it looks like typo when you are signing

Reply:
It is a string that user will copy&paste. If e.g. was copied as two parts, prefix and later address, the whitespace could cause some confusion. It is imho more visible and clear not to have it than have it. Comes down to personal preference I guess. Don't mind either way